pkgs/tools/security/sudo/default.nix

```diff
@@ -71,7 +71,7 @@ stdenv.mkDerivation rec {
 
   passthru.tests = { inherit (nixosTests) sudo; };
 
-  meta = {
+  meta = with lib; {
     description = "A command to run commands as root";
 
```

Comment:
Don't you think we can also throw out the extra new lines?

Reply:
Done.